tests/integration/test_dbapi_integration.py

```diff
@@ -885,6 +897,12 @@ def test_eager_loading_cursor_description(trino_connection):
     assert all([a == e] for a, e in zip(description_after, description_expected))
 
 
+def test_catalogs(trino_connection):
```

Comment:
What's the purpose of this test? Can you add assertion?

Reply:
Sorry that was a trace of a test I did. I have removed it.